__tests__/app.test.js

```diff
@@ -122,4 +121,39 @@ describe("/api/articles", () => {
     test("404: when there's a typo in the url", () => {
       return request(app).get("/api/bananas").expect(404);
     });
-})
\ No newline at end of file
+})
+describe("/api/articles/:article_id/comments", () => {
+  test("200: responds with all relevant comments", () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({ body }) => {
+      console.log(body)
+      expect(body.comments.length).toBe(11)
+        body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id")
+          expect(comment).toHaveProperty("votes")
+          expect(comment).toHaveProperty("created_at")
+          expect(comment).toHaveProperty("author")
+          expect(comment).toHaveProperty("body")
+          expect(comment).toHaveProperty("article_id")
+        })
+      })
+    })
+    test("400: throws error when given a bad article id", () => {
+      return request(app)
+      .get("/api/articles/lizzie/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Sorry, try again!")
+      })
+    })
+    test("404: throws error when article id is correct type but not on the table", () => {
+      return request(app)
+      .get("/api/articles/500/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.err.msg).toBe("Sorry, I couldn't find that!")
+      })
```

Comment:
We're just missing one more test here. What would happen if we sent a request for a valid ID that had no comments, i.e. article 2?